tslint.json

```diff
@@ -168,6 +168,9 @@
     "use-primitive-type": true,
     "no-map-without-usage": true,
     "rxjs-finnish": true,
-    "rxjs-no-ignored-error": false
+    "rxjs-no-ignored-error": false,
+    "ban-ts-ignore": false,
+    "no-default-import": false,
+    "callable-types": false
```

Comment:
I think this one we can leave on

Reply:
Will remove. 

I added it for this case:

![image](https://user-images.githubusercontent.com/39564353/50298183-5eb0d080-0433-11e9-99c1-bc88d2411130.png)